src/IoC/ServiceExtensions.cs

```diff
@@ -14,19 +17,28 @@
 
     public static class ServiceExtensions
     {
+        public static IServiceCollection AddServices(this IServiceCollection services)
+        {
+            return services
+                .AddTransient<IReportingHelper, ReportingHelper>()
```

Comment:
I've been favouring .AddScoped for dependencies that should have the lifetime of the current request... although I wonder if this one can be a singleton if its just some utility code that doesn't require any context or scope specific state